Travis james

james-garriss opened the pull request:
My first pull request for a Hello World pipeline.

james-garriss committed:
added Travis YAML

james-garriss committed:
Minor change

james-garriss committed:
added email notifications

james-garriss committed:
Added linter

james-garriss committed:
Merge branch 'master' into travis-james

aaronlippold commented on .travis.yml:
```diff
@@ -0,0 +1,15 @@
+notifications:
+  email:
+    recipients:
+      - jgarriss@mitre.org
```
I think that `inspec@mitre.org` is a working email but we should double check and consider making a few more focused email workgroup names we can use for the team and apps in general.

aaronlippold reviewed (changes requested):
Do we want to keep updating this a bit more before we push it formally? Perhaps adding an actual Jest test or Mocca or whatever we will be really using?

lukemalinowski committed:
CI: Add npm run test to travis

This is in preparation for merging to dev, where tests are defined.

Signed-off-by: Luke Malinowski <lmalinowski@mitre.org>

lukemalinowski committed:
CI: Add build stage to CI script

Added a new stage to .travis.yml PR #80

Signed-off-by: Luke Malinowski <lmalinowski@mitre.org>

lukemalinowski committed:
CI: adjust build stage so that it runs under test

Run Build stage under test, as we are using it to test if the
project builds, and not doing the final build.

Signed-off-by: Luke Malinowski <lmalinowski@mitre.org>

lukemalinowski committed:
Revert "CI: adjust build stage so that it runs under test"

Travis file was ill formatted

This reverts commit 818bf2c47014e11a47c0464956f1de770a25d396.
